proposals/ard_normalized_radar_backscatter.json

```
@@ -0,0 +1,122 @@
+    "categories": [
+        "cubes",
+        "sar",
+        "ard"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "The source data cube containing SAR input.",
+            "schema": {
+                "subtype": "raster-cube",
+                "type": "object"
+            }
+        },
+        {
+            "name": "elevation_model",
+            "description": "The digital elevation model to use. Set to `null` (the default) to allow the back-end to choose, which will improve portability, but reduce reproducibility.",
+            "optional": true,
+            "default": null,
+            "schema": [
+                {
+                    "type": "string",
+                    "subtype": "collection-id"
+                },
+                {
+                    "type": "null"
+                }
+            ]
+        },
+        {
+            "name": "ellipsoid_incidence_angle",
+            "description": "If set to `true`, a ellipsoidal incidence angle band is added. The values are given in degrees.",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "gamma_sigma",
+            "description": "If set to `true`, a band is added that provides the ratio of the integrated area in the Gamma projection over the integrated area in the Sigma projection (ground).",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "date_offset",
+            "description": "If set to `true`, a band is added that provides the acquisition date for each pixel. Values are day offset to the first observation date (in UTC) as specified in the metadata.",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "noise_removal",
+            "description": "If set to `false`, no noise removal is applied. Defaults to `true`, which removes noise.",
+            "optional": true,
+            "default": true,
+            "schema": {
+                "type": "boolean"
+            }
+        }
+    ],
+    "returns": {
+        "description": "Backscatter values corresponding to the chosen backscatter coefficient. The data returned is CARD4L compliant and contains metadata.\n\nBy default the backscatter values are given in linear scale. To convert them to decibel scale (dB) use the processes ``multiply()`` and ``log()``.",
```

I was not sure to which extent to speak about the dB conversion.

We could (1) go the way given above or choose to make it either 

(2) very concrete
```suggestion
        "description": "Backscatter values corresponding to the chosen backscatter coefficient. The data returned is CARD4L compliant and contains metadata.\n\nBy default, the backscatter values are given in linear scale. To convert them to decibel scale (dB) use the processes ``multiply()`` and ``log()``: *dB = multiply(10, log(linear, 10)*.",
```

or (3) not care about it at all and assume it's common knowledge:
```suggestion
        "description": "Backscatter values corresponding to the chosen backscatter coefficient. By default, the values are given in linear scale. The data returned is CARD4L compliant and contains metadata.",
```

---

I forgot that it also needs `apply`, of course. I've removed the dB wording for now, hoping that people know what to do to convert from linear to dB